src/System.Diagnostics.PerformanceCounter/src/misc/EnvironmentHelpers.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Security;
+using System.Security.Principal;
+
+namespace System
+{
+    internal static class EnvironmentHelpers
+    {
+        private static volatile bool s_isAppContainerProcess;
+        private static volatile bool s_isAppContainerProcessInitalized;
+        internal const int TokenIsAppContainer = 29;
```

Comment:
private?

Reply:
better still, yo ucan reuse TOKEN_INFORMATION_CLASS.TokenIsAppContainer out of $(CommonPath)\Interop\Windows\advapi32\Interop.TOKEN_INFORMATION_CLASS.cs